python/tvm/topi/random/kernel.py

```diff
@@ -234,6 +242,18 @@ def gen_ir(gen_ptr, out_gen_ptr, out_array_ptr):
         out_gen = irb.buffer_ptr(out_gen_ptr)
         out_array = irb.buffer_ptr(out_array_ptr)
 
+        # Check that unsigned arithmetic wraps, as it is required to implement threefry correctly.
+        irb.emit(
```

Comment:
echoing Tianqi's comment, will compilation fail on a target that doesn't support `AssertStmt` but would in theory have everything needed for Threefry PRNG?

Reply:
@tqchen If the platform does not support assert, does it fail to compile or does it just not include the assert?

Reply:
The behavior can be backend dependent, I think right now the assert will be omitted

Reply:
That should be fine then.